src/Compilers/VisualBasic/Test/Symbol/CrossLanguageTest.vb

```diff
@@ -0,0 +1,52 @@
+﻿' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+Namespace Microsoft.CodeAnalysis.VisualBasic.UnitTests
+
+    Public Class CrossLanguageTest
+        Inherits BasicTestBase
+
+        <Fact>
+        Public Sub CanAchieveHandledEvents()
+            Dim csharpSource = <![CDATA[
+[assembly:System.CLSCompliant(true)]
+public class Sample
+{
+    public void Create()
+    {
+    }
+}
+]]>
+
+            Dim csharpCompilation = CreateCSharpCompilation(
+                csharpSource,
+                compilationOptions:=New CSharp.CSharpCompilationOptions(OutputKind.DynamicallyLinkedLibrary),
+                assemblyName:="Test",
+                referencedAssemblies:={MscorlibRef_v4_0_30316_17626})
+
+            Dim method = csharpCompilation.GetTypeByMetadataName("Sample").GetMembers("Create").OfType(Of IMethodSymbol).SingleOrDefault()
+
+            Assert.NotNull(method)
+            Assert.Empty(method.HandledEvents())
+
+            Dim basicCompilation = CreateCompilationWithMscorlib40(
+<compilation>
+    <file name="a.vb">
+Imports System
+Class Sample
+    Public Event Created As EventHandler
+
+    Public Sub Create() Handles Me.Created
+    End Sub
+End Class
+    </file>
+</compilation>)
+
+            method = basicCompilation.GetTypeByMetadataName("Sample").GetMembers("Create").OfType(Of IMethodSymbol).SingleOrDefault()
+            Assert.NotNull(method)
+            Assert.Equal(method.HandledEvents().Length, 1)
```

Comment:
>Assert.Equal(method.HandledEvents().Length, 1) [](start = 12, length = 46)

I believe the expected value should come first.

Reply:
@AlekseyTs, in the case that this test is check doesn't matter what is the value, just that `HandledEvents` doesn't fail with `NRE` for csharp compilation and return something non empty for basic compilation.

Reply:
> in the case that this test checks doesn't matter what is the value, just that HandledEvents doesn't fail with NRE for csharp compilation and returns something non empty for basic compilation.

The order of arguments matters. Should the assert fail at some point in the future, the message describing the failure will likely be confusing.  

---
In reply to: [218188780](https://github.com/dotnet/roslyn/pull/29926#discussion_r218188780) [](ancestors = 218188780)

Reply:
@AlekseyTs, my apologies, I misunderstood what did you mean under
>I believe the expected value should come first.

I supposed that you suggest to check the value of received `HandledEvent`. Changed order.